### PR TITLE
[dev-qt/qtquickcontrols2] New package.

### PR DESCRIPTION
**Pull request by tbrethom**
Loosely based on the existing qtquickcontrols package.

Please feel free to comment.

Package-Manager: portage-2.3.0

### CODE REVIEW

File: dev-qt/qtquickcontrols2/qtquickcontrols2-5.7.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+inherit qt5-build
+
+DESCRIPTION="Set of next generation controls used in conjunction with Qt Quick to build complete interfaces"
+
+if [[ ${QT5_BUILD_TYPE} == release ]]; then
+	KEYWORDS="~amd64 ~arm ~arm64 ~hppa ~ppc64 ~x86"
```
Comment (Pesa) [inline dev-qt/qtquickcontrols2/qtquickcontrols2-5.7.0.ebuild]: it's a new package so start with just `amd64` (and maybe `x86`)

File: dev-qt/qtquickcontrols2/qtquickcontrols2-5.7.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+fi
+
+DEPEND="
+	~dev-qt/qtcore-${PV}
+	~dev-qt/qtdeclarative-${PV}
+	~dev-qt/qtgui-${PV}
+	~dev-qt/qtgraphicaleffects-${PV}
```
Comment (Pesa) [inline dev-qt/qtquickcontrols2/qtquickcontrols2-5.7.0.ebuild]: sort alphabetically

File: dev-qt/qtquickcontrols2/qtquickcontrols2-5.7.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+"
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+
+	qt5-build_src_prepare
+}
```
Comment (Pesa) [inline dev-qt/qtquickcontrols2/qtquickcontrols2-5.7.0.ebuild]: the whole `src_prepare` definition is redundant

File: dev-qt/qtquickcontrols2/qtquickcontrols2-5.7.0.ebuild [hunk 4]
```diff
@@ -5,7 +5,7 @@
 EAPI=6
 inherit qt5-build
 
-DESCRIPTION="Set of next generation controls used in conjunction with Qt Quick to build complete interfaces"
+DESCRIPTION="Set of next generation Qt Quick controls for the Qt5 framework"
```
Comment (Pesa) [inline dev-qt/qtquickcontrols2/qtquickcontrols2-5.7.0.ebuild]: squash this commit into the first one please

**Comment by Pesa** [comment]
Thanks! This was on my TODO list for 5.7 :-)

**Comment by tbrethom** [comment]
Fixed, thanks for your advice :)

**Comment by Pesa** [comment]
Merged, thanks!

**Pull request closed without merging**